en_US.json

```diff
@@ -2512,7 +2512,7 @@
 	"user.invite.privacy.required": "Please read and accept the policies.",
 	"user.invite.privacy.terms": "",
 	"user.invite.privacy.terms.label": "Terms of Service",
-	"user.invite.privacy.text": "I accept the {name} {privacy}, {aup}, {terms}, and {cookie}.",
+	"user.invite.privacy.text": "I accept the {name} {privacy}, {aup}, {terms} and {cookie}.",
```

Comment:
Any corresponding changes to make in other language files?

Reply:
Other languages are not applicable.